app/styles/partials/event-card.scss

```diff
@@ -26,6 +26,11 @@
     white-space: nowrap;
     width: 90%;
   }
+
+  .link.item {
+    color: #555 !important;
```

Comment:
That changes every link item's color

Reply:
it's changing only session-card, and event-card link color as it is used inside `.extra.content`.